content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -9,13 +9,14 @@ weight: 20
 <!-- overview -->
 
 This page explains how to upgrade a Kubernetes cluster created with kubeadm from version
-{{< skew latestVersionAddMinor -1 >}}.x to version {{< skew latestVersion >}}.x, and from version
-{{< skew latestVersion >}}.x to {{< skew latestVersion >}}.y (where `y > x`). Skipping MINOR versions
+{{< skew currentVersionAddMinor -1 >}}.x to version {{< skew currentVersion >}}.x, and from version
+{{< skew currentVersion >}}.x to {{< skew currentVersion >}}.y (where `y > x`). Skipping MINOR versions
 when upgrading is unsupported.
 
-To see information about upgrading clusters created using older versions of kubeadm,
-please refer to following pages instead:
+To see information about upgrading clusters created using other versions of kubeadm,
+please refer to the following list:
 
+- [Upgrading a kubeadm cluster from {{< skew latestVersionAddMinor -1 >}} to {{< skew latestVersionAddMinor 0 >}}](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
```

Comment:
on top we already have 

> {{< skew currentVersionAddMinor -1 >}}.x to version {{< skew currentVersion >}}.x, and from version

as this page is already about N-1 -> N upgrade.

Reply:
We'll backport this change, so if you're looking at docs for an older version it's (IMO) useful to see the set of links to all the available upgrade guides. If we _don't_ do that the change becomes harder to backport because we have to conditionally remove links to unsupported upgrade guides.

Reply:
i think the list on the page should be about older versions, which was the original intention:
```
- [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -2 >}} to {{< skew currentVersionAddMinor -1 >}}](https://v{{< skew currentVersionAddMinor -1 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
- [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -3 >}} to {{< skew currentVersionAddMinor -2 >}}](https://v{{< skew currentVersionAddMinor -2 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
- [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -4 >}} to {{< skew currentVersionAddMinor -3 >}}](https://v{{< skew currentVersionAddMinor -3 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
- [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -5 >}} to {{< skew currentVersionAddMinor -4 >}}](https://v{{< skew currentVersionAddMinor -4 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
```

this is backportable.

if the user wishes to see newer versions, the can use the dropdown menu with versions to navigate.
or just go to `kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/` which would have the latest list.

Reply:
It's backportable if we're happy to include broken links (I'm not - I don't want the stream of GitHub issues it implies). Can you see why there would be broken links for the backports?

Reply:
no, can you showcase an example?

Reply:
Ah, I get it. We'll need to think about to handle this for the first release that goes out after we merge this change.